tmk_core/rules.mk

```diff
@@ -331,7 +331,7 @@ $1/%.o : %.S $1/asflags.txt $1/compiler.txt | $(BEGIN)
 $1/%.a : $1/%.o
 	@mkdir -p $$(@D)
 	@$(SILENT) || printf "Archiving: $$<" | $$(AWK_CMD)
-	$$(eval CMD=$$(AR) $$@ $$<)
+	$$(eval CMD=$$(AR) $$(ARFLAGS) $$@ $$<)
```

Comment:
`$$(eval CMD=$$(AR) rcs $$@ $$<)`

This should be explicit because "rcs" is the operation this line is performing. Other `ar` invocations elsewhere could perform other operations, so it doesn't make sense to put it in a global `$ARFLAGS` variable.

Reply:
OK Let's do that.

(However, I have been using ar for many years, have never used an operation other than 'r' myself, and I have not seen it in other makefiles. :-)

Reply:
Oh, absolutely. It's entirely me being pedantic about semantics :)